docs/page-objects/testing.rst

```diff
@@ -217,3 +217,24 @@ Please also check the official Git LFS documentation for more information.
 
 .. _Git LFS: https://git-lfs.com/
 .. _implementations: https://github.com/git-lfs/git-lfs/wiki/Implementations
+
+Additional requests support
+===========================
+
+If the page object uses the :class:`~.HttpClient` dependency to make
+:ref:`additional requests <additional-requests>`, the generated fixtures will
+contain these requests and their responses. When the test runs,
+:class:`~.HttpClient` will return the saved responses without doing actual
+requests.
+
+Currently requests are compared by their URL, method, headers and body, so if a
+page object makes requests that differ between runs, the test won't be able to
+find a saved response and will fail.
```

Comment:
It might also be noteworthy to mention that these are saved with the following filenames:

*   `HttpClient-n-HttpRequest.info.json`
*   `HttpClient-n-HttpResponse.body.html`

Or maybe show them in the GNU tree output in the fixtures section above.